backend/src/services/pattern_extraction_service.py

```
@@ -1,134 +1,698 @@
-Pattern Extraction Service
+Pattern Extraction Service for intelligent merchant recognition and category suggestions.
 
-This service handles intelligent pattern extraction from transaction descriptions,
-merchant recognition, and category name suggestions for Phase 4.1.
+This service provides smart pattern extraction from transaction descriptions,
+merchant recognition, and category name suggestions for quick categorization workflows.
 """
 
 import logging
 import re
 import uuid
```

Ruff (0.11.9)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)